scripts/dockerhost/start-dockerhosts

```diff
@@ -1,11 +1,40 @@
 #!/bin/bash
 
+set -e
+
 scriptdir=`dirname "$BASH_SOURCE"`
 echo $scriptdir
 
+if ! which nsenter 2>&1 >/dev/null
+then
+  echo "Please install nsenter. It's a part of the util-linux package, which may not be current on your system."
+  exit 1
+fi
+
+if ! sudo ip link | grep -q docker0
+then
```

Comment:
presence of docker0 may not be the best way to detect if docker daemon is running. perhaps we can use `service docker status` to confirm that. With netplugin some folks may disable docker0 bridge all together.